src/frontend/src/lib/components/tokens/TokensSignedOut.svelte

```diff
@@ -8,7 +8,7 @@
 	onMount(unsafeLoadDefaultPublicIcrcTokens);
 </script>
 
-{#each $enabledNetworkTokens as token (token.id)}
+{#each $combinedDerivedSortedNetworkTokens as token (token.id)}
```

Comment:
I'm not sure why you change this. It seems unrelated to the PR fix and it's also the list of tokens displayed when the user is not signed-in - i.e. no exchange values should be fetched at that time. What's the reason?

Reply:
before this PR, since `enabledNetworkTokens` derived, after some level, from `combinedDerivedSortedTokens`, that list was sorted according to the "sorting logic". Meaning that we were showing the signed-out list in a coherent order in respect to the signed in one (even if blurred).

After this PR that would have not be true anymore: `combinedDerivedSortedNetworkTokens` becomes derived from `enabledNetworkTokens`, with the only difference of the pinned tokens and the sorted ones.

Reply:
Ah thanks for the explanation. Somehow I was suprised that the "Signed-out" tokens list used a store that relies on the exchanges values.